File: webservice/Frikollection_Api/Frikollection_Api/DTOs/Collection/CollectionStatsDto.cs
namespace Frikollection_Api.DTOs.Collection
{
    public class CollectionStatsDto
    {
        public Guid CollectionId { get; set; }
        public int TotalProducts { get; set; }
        public decimal TotalValue { get; set; }
        public Dictionary<string, int> ProductTypes { get; set; } = new();
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/collection/CollectionStatsDto.kt
package org.example.frikollection_mobile_desktop.models.collection

import kotlinx.serialization.Serializable

@Serializable
data class CollectionStatsDto(
    val collectionId: String,
    val totalProducts: Int,
    val totalValue: Double,
    val productTypes: Map<String, Int>
)